etc/config/c++.amazon.properties

```diff
@@ -1101,6 +1101,14 @@ libs.gsl.versions.301.version=3.0.1
 libs.gsl.versions.301.path=/opt/compiler-explorer/libs/GSL/v3.0.1/include
 libs.gsl.versions.lite.version=lite
 libs.gsl.versions.lite.path=/opt/compiler-explorer/libs/gsl-lite/include
+libs.entt.name=entt
+libs.entt.versions=trunk:350:360
+libs.entt.versions.350.version=v3.5.0
```

Comment:
Do you want the `v` in the version? Most libraries go with just `3.5.0` etc here - the `v` is a github thing usually

Reply:
Will merge and deploy and if you prefer the name to be `3.5.0` let me know

Reply:
You're right, I just added the v because it was in the tag name. 
I didn't realize it was just a display name. "3.5.0" seems more consistent with the rest of the libraries.

Reply:
Will do! It'll be picked up in a subsequent site update. Thanks again!